Userland/Libraries/LibPDF/ColorSpace.cpp

```diff
@@ -229,7 +229,7 @@ PDFErrorOr<NonnullRefPtr<CalRGBColorSpace>> CalRGBColorSpace::create(Document* d
 
     if (dict->contains(CommonNames::Matrix)) {
         auto matrix_array = TRY(dict->get_array(document, CommonNames::Matrix));
-        if (matrix_array->size() == 3) {
+        if (matrix_array->size() == 9) {
```

Comment:
Feels like this mistake would be further avoidable with:
```c++
if (matrix_array->size() == color_space->m_matrix.size())
```
(and similar for size checks above)